tests/test_api.py

```diff
@@ -2141,45 +2141,6 @@ def test_record_queries(self, record_query_mock: MagicMock) -> None:
         assert metadata["dataset"] == "events"
         assert metadata["request"]["referrer"] == "test"
 
-    @patch("snuba.web.query._run_query_pipeline")
-    @patch("snuba.web.query._run_new_query_pipeline")
-    def test_error_handler(
```

Comment:
what was the point of this test?

Reply:
it already exists in `test_snql_api` it's a duplicate